src/elm/Util/PhoneNumber.elm

```diff
@@ -19,6 +19,9 @@ withDefaultCountryCode phone =
     if String.isEmpty phone then
         phone
 
+    else if String.endsWith "+47" phone then
```

Comment:
Mulig dette burde være mer generelt for å håndtere flere landskoder. I App-en er vel dette to felt, og vi kan vurdere å gjøre det samme her, men det krever oversikt over kodene. Det endrer seg sjeldent så ikke et stort problem.

Tenker dette gjør det litt enklere for mesteparten av brukere, så en grei løsning i mellomtiden.